shared/testutil/assert/assertions.go

```diff
@@ -0,0 +1,30 @@
+package assert
```

Comment:
What is wrong with just creating `testutil.AssertEqual` instead of having a whole separate package for this ?

Reply:
There's definitely nothing wrong with using `testutil.AssertEqual`, I've just assumed that `assert.Equal` is a bit more conventional, reads better and it is a tad shorter. However, I do not have strong feelings about that - so, if you believe that new packages shouldn't have been created, I can move asserts/requires into the same `testutil` package.

Reply:
Creating separate packages just for assert/require does seem a bit of an overkill, but I dont have a too strong opinion on it. If you think it helps in reading it better then its fine.